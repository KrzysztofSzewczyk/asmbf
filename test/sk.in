((((S(KS))((S(K(S(KS))))((S(K(S(KK))))((SK)K))))((SK)K))((SK)K))
