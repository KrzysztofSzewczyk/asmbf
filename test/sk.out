((S((S(KS))((S(KK))((SK)K))))((SK)K))